pyproject.toml

```diff
@@ -55,6 +55,7 @@ sphinx-copybutton = "*"
 sphinx-rtd-theme = "*"
 sphinxcontrib-bibtex = "*"
 sphinxcontrib-napoleon = "*"
+sphinx_tabs = "*"
```

Comment:
I recommend separating development dependencies from documentation dependencies, especially if we are not including running examples in function's docs. In such cases, installing development dependencies during the documentation build is unnecessary and can increase the doc's build time.